packages/compass/package.json

```diff
@@ -182,7 +182,7 @@
     "@mongodb-js/compass-database": "^3.2.0",
     "@mongodb-js/compass-databases-collections": "^1.2.0",
     "@mongodb-js/compass-explain-plan": "^6.2.0",
-    "@mongodb-js/compass-export-to-language": "^8.2.0",
+    "@mongodb-js/compass-export-to-language": "^0.1.0",
```

Comment:
We probably want the old version to be preserved

Reply:
aah good point. I think I miss that every time :)

Reply:
Yeah, trips me up all the time too 😅